### ISSUE TITLE
Is HoloLens support incomplete?

### ISSUE THREAD

**Issue by Noemata** [RolandKoenig/SeeingSharp#1]
When running the HoloLens3DTest, the app dies here:

        public RenderLoop RenderLoop
        {
            get
            {
                throw new NotImplementedException();
            }
        }

Presumably because this really hasn't been implemented?

Is this work in progress?

**Comment by RolandKoenig** [comment]
No, it's not implemented and currently not in progress because of my master study. Sorry
